### ISSUE TITLE
Missing Alt Text for Gallery Images

### ISSUE THREAD

**Issue by GauravKarakoti** [GauravKarakoti/Omnifood#1]
Gallery images in `index.html` have non-descriptive alt text (e.g., "Photo of beautifully arranged food"). Update alt text to be more descriptive for screen readers.

**Comment by Th3Silv4n3lf** [comment]
Can i have this issue? @GauravKarakoti

**Comment by IshanOstwal** [comment]
Can I have this Issue? @GauravKarakoti

**Comment by GauravKarakoti** [comment]
> Can I have this Issue? [@GauravKarakoti](https://github.com/GauravKarakoti)

Go ahead @IshanOstwal
